bin/clang/pragmas.cc

```diff
@@ -340,14 +343,24 @@ SSTPragma::getMap(SourceLocation loc, const std::list<clang::Token>& tokens)
         argList.push_back(sstr.str());
         sstr.str("");
       } else {
-        tokenToString(t, sstr);
+        sstr << ",";
       }
       break;
     default:
+      if (!hitParen && sstr.tellp() != 0){
+        allArgs[sstr.str()] = std::list<std::string>();
+        sstr.str("");
+      }
       tokenToString(t, sstr);
       break;
     }
   }
+  //trailing standalone argument
+  if (sstr.tellp() != 0){
+    allArgs[sstr.str()] = std::list<std::string>();
```

Comment:
map operator[], already default constructs the list, unless I am missing something and you meant to overwrite it with an empty list.

Reply:
this is just to make things easier to read